src/BioStructures.jl

```diff
@@ -25,11 +25,14 @@ using Graphs
 using MetaGraphs
 import MMTF: parsemmtf, writemmtf # Imported to avoid clash with MMTF name
 using RecipesBase
+using STRIDE_jll: STRIDE_jll # for secondary structure prediction
```

Comment:
Stride is an alternative to DSSP. I added it because [ProteinSecondaryStructures.jl](https://github.com/m3g/ProteinSecondaryStructures.jl) supports both of them.

Reply:
I meant writing `using STRIDE_jll: STRIDE_jll` rather than `using STRIDE_jll`. Maybe it is required.

Reply:
This is not required, I will remove it.